data/chameleoncloud/sites/uc/clusters/chameleon/nodes/1134b490-9a5d-46ac-aa61-997cf0a4cd85.json

```diff
@@ -15,10 +15,7 @@
     "serial": "GNHFDX3"
   },
   "gpu": {
-    "gpu": true,
-    "gpu_count": 1,
-    "gpu_model": "GA100 [A100 PCIe 80GB]",
-    "gpu_vendor": "NVIDIA"
+    "gpu": false
   },
```

Comment:
this is good example of "current state" vs "desired state" of a node.

in our ground truth, we know gigaio-01 should have a gpu.
but we also know that inventory isn't seeing it, which indicates a problem with the node.

This is indeed ok to merge, but we should be thinking about how to detect when hardware that "should" be there has vanished.